### DISCUSSION TITLE
【Web3Con】Raise your hand to join the hackathon team!

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#47] [Announcements]
Topic: `3D Printer`、`Voxel`、`NFT`

Time: `February 28-March 6, 2022`

Informations:

> https://www.web3con.dev/hackathon

Repo:

> https://github.com/WeLightProject/Tai-Shang-Voxel-Handler/

**Comment by g302ge** [comment]
🙋‍♂️

**Comment by Dream4ever** [comment]
🙋‍♂️
继续凑热闹，哈哈~

**Comment by 0xKT** [comment]
🙋‍♂️

**Comment by fewwwww** [comment]
🙋‍♂️

**Comment by Lingyue1213** [comment]
🙋‍♂️

**Comment by leeduckgo** [comment]
<img width="870" alt="image" src="https://user-images.githubusercontent.com/12784118/156108807-a96f7bfb-b9d3-49be-a0fc-a1db4a649fe0.png">
